books/RayTracingTheRestOfYourLife.html

```diff
@@ -784,12 +787,12 @@
 PDF. Now we need to figure out a way to solve for $f(d)$.
 
 For the PDF given above, where $p(r) = \frac{r}{2}$, the probability of a random sample is higher
-toward 2 than it is toward 0. More 1.8s and 1.9s should be generated than 0.1s and 0.2s. If we put
-aside our mathematics hat for a second and put on our computer science hat, maybe we can figure out
-a smart way of partitioning the PDF. We know that there is a higher probability near 2 than near
-0, but what is the value that splits the probability in half? What is the value that a number chosen
-at random has a 50% chance of being higher than and 50% chance of being lower than? What is the $x$
-that solves:
+toward 2 than it is toward 0. There is a greater probability of getting a number betwween 1.8 and
+2 than 0 and 0.2. If we put aside our mathematics hat for a second and put on our computer
```

Comment:
“... between [note typo] 1.8 and 2.0 than there is between 0.0 and 0.2.”

Reply:
That was deliberate, but I'll fix it.